integrations/system/disk-monitoring/README.md

```diff
@@ -0,0 +1,113 @@
+## Overview
+
+<!-- Sensu Integration description; supports markdown -->
+
+The disk monitoring integration integration provides disk usage monitoring for Linux, Windows, and macOS system.
+
+<!-- Provide a high level overview of the integration contents (e.g. checks, filters, mutators, handlers, assets, etc) -->
+
+This integration provides the following resources:
+
+* "disk-usage" [check]
+* "sensu/check-disk-usage:0.6.0" [asset]
```

Comment:
Small thing..
Should be we adding the versioned asset name here as best practice?  I think so far we've been inconsistent with this in the first batch of integrations.

Reply:
I don't think this is a deal breaker one way or the other, but let's go with versioned references for now.